features/data_converter/binary_protobuf/feature.go

```diff
@@ -0,0 +1,63 @@
+package binary_protobuf
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/gogo/protobuf/proto"
+	pb "go.temporal.io/features/features/data_converter"
+	"go.temporal.io/features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/sdk/workflow"
+)
+
+var EXPECTED_RESULT = []byte{0xde, 0xad, 0xbe, 0xef}
+
+var Feature = harness.Feature{
+	Workflows:   Workflow,
+	CheckResult: CheckResult,
+	ClientOptions: client.Options{
+		DataConverter: converter.NewCompositeDataConverter(
+			converter.NewNilPayloadConverter(),
+			// Disable ByteSlice and ProtoJSON converters
+			converter.NewProtoPayloadConverter(),
+			converter.NewJSONPayloadConverter(),
+		),
+	},
+}
+
+func Workflow(ctx workflow.Context) (pb.BinaryMessage, error) {
```

Comment:
The default `runner.Execute` does not take arguments for a workflow. We could do it with a custom `Execute`, or a child workflow. I think both would complicate the feature though, and with child workflows you always have the doubt of whether it only applies to child workflows... 

I'm leaning  towards opening an issue to add default workflow arguments in ExecuteDefault, and then get back to this when it is supported...

Reply:
I think you just happen to be the first person to need arguments for a workflow. No need to wait for another issue. Can do this generically, but I think something like this is simple enough (unchecked just typed here, so may have typos):

```
Execute: func(ctx context.Context, r *Runner) (client.WorkflowRun, error) {
    opts := r.Feature.StartWorkflowOptions
    opts.TaskQueue = r.TaskQueue
    return r.Client.ExecuteWorkflow(ctx, opts, Workflow, &MyWorkflowParam{})
},
```

That's only a few lines.

Reply:
Adding an echo workflow to test in both directions. 
Replay was failing with a custom codec because the DataConverter option was not propagated to WorkflowReplay, fixed in runner.go, please check.
It was a great idea to add this extra check, this was not showing up before...